checkout/checkout_service.bal

```diff
@@ -110,17 +119,11 @@ service "CheckoutService" on new grpc:Listener(9094) {
             items: orderItems
         };
 
-        error? err = self.sendConfirmationMail(request.email, 'order);
-        if err is error {
-            log:printWarn(string `failed to send order confirmation to ${request.email}`, 'error = err);
-        } else {
-            log:printInfo(string `order confirmation email sent to ${request.email}`);
-        }
-
+        check self.sendConfirmationMail(request.email, 'order);
```

Comment:
original example doesn't return an error to client, instead logs

Reply:
https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/src/checkoutservice/main.go#L277-L281

Reply:
Updated in https://github.com/ballerina-guides/gcp-microservices-demo/pull/30/commits/ea21987e734905602ec7a060273e128c6015493d